charts/icm-as/templates/as-deployment.yaml

```diff
@@ -156,6 +156,11 @@ spec:
               protocol: TCP
           resources:
             {{- toYaml .Values.resources | trim | nindent 12 }}
+          lifecycle:
+            preStop:
+              httpGet:
+                port: 7743
+                path: /servlet/Hook/Shutdown
```

Comment:
The URL should be more semantically structured, not implementation related . "/servlet/admin/shutdown" would be a better name.

Reply:
That's a good point, I renamed the URL to be more semantic.